mood_bot.py

```diff
@@ -0,0 +1,726 @@
+                'lonely', 'prideful', 'confused', 'disgusted', 'surprised', 'disappointed')
+
+"""Power is for switching the auto_checker ON/OFF"""
+auto_checker_power_switch = 'OFF'
+
+""" Gates are for the mood check execution; to make sure that the mood check is done in the correct order. """
+date_gate = 'Open'
+type_gate = 'Closed'
+rating_gate = 'Closed'
+why_gate = 'Closed'
+stage_array = ['date', 'type', 'rating', 'explanation']
+
+"""Repeated Strings"""
+day_format_error_message = "Please enter a valid day of the week or a term like 'Everyday' or 'Weekdays' or 'Weekends'."
+time_duplicate_error_message = 'Time is already in schedule.'
+time_format_error_message = 'Please write time in HH:MM format.'
+auto_check_input_error_message = 'Please specify whether the auto checker is ON or OFF.'
+start_mood_check_message = 'Please start a mood check first.'
+mood_type_error_message = "Please enter in one of the following mood types:\n" + str(mood_choices)
+mood_type_message = 'You left off at the mood type.'
+rating_error_message = 'Please enter an integer from 1 to 10.'
+integer_message_for_diary = 'Please use integers to specify the start and end entries.'
+diary_empty_message = 'Diary is empty. Please do a mood check to add an entry.'
+out_of_diary_range_message_partial = 'For start and end entries, please specify integers from 1 to '
+
+""" MOOD CALENDAR FUNCTIONS """
+
+
+def num_days_to_word_days(days):
+    word_days = []
+    if days == [1, 1, 1, 1, 1, 1, 1]:
+        word_days = 'everyday'
+    elif days == [1, 1, 1, 1, 1, 0, 0]:
+        word_days = 'weekdays'
+    elif days == [0, 0, 0, 0, 0, 1, 1]:
+        word_days = 'weekends'
+    else:
+        for num, day in enumerate(days):
+            if day == 1:
+                word_days.append(days_of_week[num])
+    return word_days
+
+
+async def day_format_error(context, day_input):
+    await context.send(day_format_error_message)
+    raise me.InputMoodError(day_format_error_message, day_input)
+
+
+async def time_format_error(context, time_input):
+    await context.send(time_format_error_message)
+    raise me.InputMoodError(time_format_error_message, time_input)
+
+
+async def duplicate_time_error(context, time_input):
+    await context.send(time_duplicate_error_message)
+    raise me.InputMoodError(time_duplicate_error_message, time_input)
+
+
+async def new_schedule_display_message(context, days_or_times, new_sched):
+    await context.send(f"The {days_or_times} in your mood schedule are now:\n{new_sched}")
+
+
+@mood_bot.command(name='add_day', help='Add day(s) to mood schedule.',
+                  aliases=['addday', '+day', 'day_add', 'dayadd', '+days', 'add_days', 'adddays'])
+async def add_day(context, *day_inputs):
+    """ This function allows the user to add a day of the week to their mood check schedule. """
+    author = context.author
+    [mood_days, _] = mood_schedule(author)
+    for day_input in day_inputs:

```

Comment:
we can `.map()` this to `.lower()`

Reply:
I googled and learned that you can do this in Python using the lambda function.
'day_inputs_lower = list(map(lambda x: x.lower(), day_inputs))'